src/main/java/com/antherd/smcrypto/NashornProvider.java

```diff
@@ -0,0 +1,71 @@
+package com.antherd.smcrypto;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * @author Ice2Faith
+ * @date 2024/8/7 19:54
+ * @desc
```

Comment:
Add descriptions or Remove this tag.
If you want to add custom annotation tags, you also need to add maven-javadoc-plugin config options,
<plugin>
    <groupId>org.apache.maven.plugins</groupId>
    <artifactId>maven-javadoc-plugin</artifactId>
    <version>2.9.1</version>
    <configuration>
        <!-- for jdk1.8, remove this on jdk1.7 -->
        <additionalJOptions>
            <additionalJOption>-Xdoclint:none</additionalJOption>
        </additionalJOptions>
    </configuration>
</plugin>